boards/discovery_f412g/README.md

```diff
@@ -0,0 +1,58 @@
+STM32F4 Discovery Kit with STM32F412G MCU
+======================================================
+
+For more details [visit STM32F412G Discovery Kit
+website](https://www.st.com/en/evaluation-tools/32f412gdiscovery.html).
+
+## Flashing the kernel
+
+The kernel can be programmed using OpenOCD. `cd` into `boards/discovery_f412g`
+directory and run:
+
+```bash
+$ make flash
+
+(or)
+
+$ make flash-debug
+```
+
+> **Note:** Unlike other Tock platforms, the default kernel image for this
+> board will clear flashed apps when the kernel is loaded. This is to support
+> the non-tockloader based app flash procedure below. To preserve loaded apps,
+> comment out the `APP_HACK` variable in `src/main.rs`.
```

Comment:
Why are all of these platforms still doing this? My impression is the naming implied this was short-term, but if it is getting copied over and over, this is going to become less a hack and more of a standard. What would it take to fix this?

Reply:
I did not implement the original Nucelo boards, I think tockloader does not support this platform so far.